apps/sap-commerce-cloud/frontend/src/AppConfig/AppConfig.tsx

```diff
@@ -39,18 +39,39 @@ interface Props {
   description: string;
 }
 
-export default function AppConfig(props: Props) {
+export default function AppConfig({
+  sdk,
+  parameterDefinitions,
+  validateParameters,
+  logo,
+  name,
+  color,
+  description,
+}: Props) {
   const [contentTypes, setContentTypes] = useState<ContentType[]>([]);
   const [compatibleFields, setCompatibleFields] = useState<CompatibleFields>({});
   const [selectedFields, setSelectedFields] = useState<SelectedFields>({});
   const [parameters, setParameters] = useState<Config>(
-    toInputParameters(props.parameterDefinitions, null)
+    toInputParameters(parameterDefinitions, null)
   );
 
+  const { space, app, ids } = sdk;
+
   useEffect(() => {
-    const init = async () => {
-      const { space, app, ids } = props.sdk;
+    const onAppConfigure = () => {
```

Comment:
moved this into the `useEffect` hook trying to resolve dependency array related issues, but just left it here since it's only called on mount via the `init` function anyway.